config/samples/enterprisesearch/ent_es.yaml

```
@@ -13,6 +13,18 @@ spec:
         # See: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-virtual-memory.html
         node.store.allow_mmap: false
 ---
+apiVersion: kibana.k8s.elastic.co/v1
+kind: Kibana
+metadata:
+  name: kibana-sample
+spec:
+  version: 8.15.0
```

The version was updated in https://github.com/elastic/cloud-on-k8s/pull/8205. 
```suggestion
  version: 8.16.0
```